Documentation/guides/messages/xa1024.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Xamarin.Android warning XA1024
+description: XA1024 warning code
+ms.date: 07/08/2020
+---
+# Xamarin.Android warning XA1024
+
+## Example messages
+
+```
+warning XA1024: Use of 'Foo.dll.config' is not supported in Xamarin.Android for .NET 5 and higher.
+```
+
+## Issue
+
+No mechanism for supporting `Foo.dll.config` or `Foo.exe.config` files
+exists in Xamarin.Android for .NET 5 and higher. [`<dllmap>`][dllmap]
+is not supported in .NET 5 at all. Compatibility packages like
+[System.Configuration.ConfigurationManager][nuget] have never been
+supported in Xamarin.Android either.
```

Comment:
One other question crossed my mind about `<assemblyBinding>` elements, but it looks like that probably doesn't need to be mentioned since I think those aren't used by Xamarin.Android legacy projects either, and it seems those elements also aren't supported in .NET 5 at all.